siriuspy/siriuspy/devices/scraper.py

```diff
@@ -0,0 +1,311 @@
+        """Go to maximum slit aperture."""
+        self['Home-Cmd'] = 1
+
+
+class ScraperH(_ScraperDev):
+    """."""
+
+    _properties = (
+        'OuterSlitPos-SP', 'OuterSlitPos-RB',
+        'InnerSlitPos-SP', 'InnerSlitPos-RB',
+        'InnerDoneMov-Mon', 'OuterDoneMov-Mon',
+        'OuterSlitInnerLim-SP', 'OuterSlitInnerLim-RB',
+        'OuterSlitOuterLim-SP', 'OuterSlitOuterLim-RB',
+        'OuterSlitBacklashDist-SP', 'OuterSlitBacklashDist-RB',
+        'OuterMotionCtrl-Cte', 'ForceOuterSlitPos-Cmd',
+        'InnerSlitInnerLim-SP', 'InnerSlitInnerLim-RB',
+        'InnerSlitOuterLim-SP', 'InnerSlitOuterLim-RB',
+        'InnerSlitBacklashDist-SP', 'InnerSlitBacklashDist-RB',
+        'InnerMotionCtrl-Cte', 'ForceInnerSlitPos-Cmd',
+        )
+
+    def __init__(self):
+        """."""
+        # call base class constructor
+        _all_properties = _ScraperDev._properties + ScraperH._properties
+        super().__init__(
+            devname=_ScraperDev.DEVICES.H, properties=_all_properties)
+
+    @property
+    def left_slit(self):
+        """Left slit position [mm]."""
+        return self['OuterSlitPos-RB']
+
+    @property
+    def right_slit(self):
+        """Right slit position [mm]."""
+        return self['InnerSlitPos-RB']
+
+    @property
+    def is_left_slit_moving(self):
+        """."""
+        return not self['OuterDoneMov-Mon']
+
+    @property
+    def is_right_slit_moving(self):
+        """."""
+        return not self['InnerDoneMov-Mon']
+
+    @property
+    def left_slit_limits(self):
+        """Limits for left slit.
+
+        Left Slit (x>0)             Right Slit (x<0)
+        [Outer, Inner] | (x=0) | [Inner, Outer]
+        """
+        return [self['OuterSlitOuterLim-RB'], self['OuterSlitInnerLim-RB']]
+
+    @left_slit_limits.setter
+    def left_slit_limits(self, limits):
+        """Set left slit limits.
+
+        Left Slit (x>0)             Right Slit (x<0)
+        [Outer, Inner] | (x=0) | [Inner, Outer]
+        """
+        self['OuterSlitInnerLim-SP'] = limits[0]
+        self['OuterSlitOuterLim-SP'] = limits[1]
+
+    @property
+    def right_slit_limits(self):
+        """Limits for right slit.
+
+        Left Slit (x>0)             Right Slit (x<0)
+        [Outer, Inner] | (x=0) | [Inner, Outer]
+        """
+        return [self['InnerSlitInnerLim-RB'], self['InnerSlitOuterLim-RB']]
+
+    @right_slit_limits.setter
+    def right_slit_limits(self, limits):
+        """Set right slit limits.
+
+        Left Slit (x>0)             Right Slit (x<0)
+        [Outer, Inner] | (x=0) | [Inner, Outer]
+        """
+        self['InnerSlitInnerLim-SP'] = limits[0]
+        self['InnerSlitOuterLim-SP'] = limits[1]
+
+    @property
+    def left_slit_control_prefix(self):
+        """."""
+        return self['OuterMotionCtrl-Cte']
+
+    @property
+    def right_slit_control_prefix(self):
+        """."""
+        return self['InnerMotionCtrl-Cte']
+
+    def cmd_move_left_slit(self, value):
```

Comment:
methods starting with `cmd_` are reserved to act in `-Cmd` PVs, hence they don't accept an input value.
Please, consider changing the prefix of these methods to something else. I think that only `move_left_slit` is ok.

Reply:
done!